karma.conf.js

```diff
@@ -37,7 +37,7 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['ChromeHeadless'],
```

Comment:
this changes the test experience for the devs in a negative way, since they don't see the browser environment of the tests, which brings several small benefits. fortunately it's possible to pass a `--browsers` argument to ng test to override this file, so you can revert the change to it.

Reply:
Thanks for pointing this out. Switched browser back to Chrome in karma.conf.js. Added --browsers argument to the unit test task.